spec/system/device_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
```

Comment:
Did you mean to name the file "devi**s**e_spec"

Reply:
resolved in https://github.com/owaisrazaeng1/nimble-test/commit/7aa833bd51593f376ce1c36a37c31e7b314e627f